docs/compute-engine/quickstart_android.md

```diff
@@ -1,19 +1,190 @@
+To include binaries only for Android ABIs you need to support, modify `abiFilters`
+in the Gradle build as described [here](https://www.tensorflow.org/lite/guide/android#use_the_tensorflow_lite_aar_from_jcenter).
+
+!!! note
+    In TensorFlow Lite image classification app, you need to also remove
+    the standard TensorFlow Lite dependency by commenting out the following
+    line in `build.gradle` file:
+
+    ```
+    dependencies {
+        // Comment out the following line
+        // implementation 'org.tensorflow:tensorflow-lite:0.0.0-nightly'
+    }
+    ```
+    Note that the dependencies `tensorflow-lite-gpu` and `tensorflow-lite-support` are still required!
+
+#### Build LCE AAR locally ####
+
+In cases that you would like to make local changes to the LCE code or
+TensorFlow Lite binaries, you might wish to build the LCE AAR locally.
+We provide a bash script to build the LCE AAR. The bash script is only
+tested inside the LCE docker container. See LCE [build guide](/compute-engine/build) to
+setup the docker container.
+
+Once the docker container is setup, run the following command from the LCE root
+directory inside the container:
+
+``` bash
+./larq_compute_engine/tflite/java/build_lce_aar.sh
+```
+The script will build the LCE Android archive 'lce-lite-\<version\>.aar` and
+store the file in the root directory of LCE repository.
+
+Transfer the LCE Android archive file from the LCE container to the host
+machine (where the Android Studio is setup).
+Execute the following command to install the LCE Android Archive
+to local Maven repository:
+
+```
+mvn install:install-file \
+  -Dfile=lce-lite-0.1.aar \
+  -DgroupId=org.tensorflow \
+  -DartifactId=lce-lite -Dversion=0.1.000 -Dpackaging=aar
+```
+
+Modify the `build.gradle` file in the the Android project to
+include `mavenLocal()` repository:
+
+```
+allprojects {
+    repositories {
+        jcenter()
+        mavenLocal()
+    }
+}
+```
+
+Replace the standard TensorFlow Lite dependency with
+local LCE AAR:
+
+```
+dependencies {
+    implementation 'org.tensorflow:lce-lite:0.1.000
+}
+```
+
+### 3. Add Larq Model to the project ###
+
+In this guide, we use the [Larq QuickNet](https://docs.larq.dev/zoo/api/sota/#quicknet)
+model for efficient and fast image classification. The FlatBuffer format of the QuickNet model
+'quicknet.tflite' (see [here](/compute-engine/converter) to convert a Larq model 
+to FlatBuffer format) needs to be placed in the `assets` folder of the Android project.
+You also needs to use the `labels_without_background.txt`  as its corresponding labels file.
+The Labels file is already available in the `asset` folder of Android project.
+
+The TensorFlow Lite classifier in the image classifaction app works with the
+float MobileNet model. To replace the MobileNet with QuickNet, you need to modify
+`getModelPath()` and `getLabelPath()` methods in `ClassifierFloatMobileNet.java`
```

Comment:
In the code I downloaded, `CameraActivity.java` defines the default as
```java
private Model model = Model.QUANTIZED_EFFICIENTNET;
```
I also think it would be nice to add a new class for QuickNet rather than modifying the `ClassifierFloatMobileNet` class, but for the purpose of this tutorial I suppose it is much easier to illustrate these two simple changes.

Reply:
I agree that it's much cleaner to have a separate class for QuickNet but as you mentioned at this point it will be overkill since the purpose of this tutorial its just showing how to integrate LCE in user's own android app. If we continue to further change the android apps in the TF `examples` repo, we might need to fork the repo at some point and publish our own versions.